src/translation-assistant.js

```diff
@@ -374,6 +374,16 @@ function rtrim(str) {
     return str.replace(/\s+$/g, '');
 }
 
+/**
```

Comment:
Lets not extend the `RegExp` object, but instead make this into a separate function, perhaps `escapeForRegex`?

Can you also please add a sentence at the top of this jsdoc comment to briefly describe what this function does, for example something like:

```
/**
 * Escape any string to create a regular expression.
 *
 * See: https://stackoverflow...
```